README.md

```diff
@@ -1,84 +1,68 @@
+
 # Welcome to MOLGENIS
 
-MOLGENIS is an collaborative open source project on a mission to generate great software infrastructure for life science research. Each app in the MOLGENIS family comes with rich data management interface and plug-in integration of analysis tools in R, Java and web services.
+MOLGENIS is a collaborative open source project on a mission to generate great software infrastructure for life science research. 
 
 The procedure below tells you how to checkout the molgenis project and build one of the example apps.
 
-## 1. Fork the molgenis repository
-
-Create a fork of the github.com/molgenis/molgenis repository for your own Github account.
-More information about forking can be found here: https://help.github.com/articles/fork-a-repo
-
-## 2. Clone your molgenis fork on your local system
-
-Pick a folder for your Git repositories, e.g.:
-
-    cd ~/git
-
-Clone your fork to this folder:
-
-    git clone https://github.com/<yourname>/molgenis.git
+## 1. Install software needed
 
-Optionally select stable molgenis version:
+install java8 JDK (not JRE) from http://www.oracle.com/technetwork/java/javase/downloads/index.html
 
-    git fetch --tags origin
-    git checkout -b <tag name: see https://github.com/molgenis/molgenis/releases>
-    
-## 3. Install and configure eclipse
+install eclipse (we used Luna) from https://www.eclipse.org/downloads/
 
-Compiling MOLGENIS requires Java 7.
-MOLGENIS is created with help of Maven and Freemarker. You need a few eclipse plugins to work with those.
+install mysql from (we used 5.6.23 DMG) http://dev.mysql.com/downloads/mysql/
 
-Important: get the LATEST version of Eclipse and all plugins mentioned. For example, using Eclipse Juno Release 1 or lower might result in infinite build loops.
+install git from http://git-scm.com/download/
 
-Start Eclipse and select a workspace location (e.g. ~/eclipse/workspace - do not pick the ~/git folder).
+## 2. Configure mysql
 
-Now install the plugins. Click: Help -> Eclipse marketplace. Add the following (restart Eclipse when done):
+open terminal and start mysql client (command differs in Windows, Linux, Mac): 
 
-* maven integration for eclipse
-* Apt M2E connector
-* JBoss Tools (ONLY SELECT THE 'FreeMarker IDE feature' later in the wizard!)
-* testng
+    /usr/local/mysql/bin/mysql -u root
 
-Search for updates and install if they exist
+in mysql client create database and permissions via command:
 
-## 4. Import the molgenis project into eclipse
-   
-Start Eclipse and select your workspace location (e.g. ~/eclipse/workspace).
+    create database omx;
+    grant all privileges on omx.* to molgenis@localhost identified by 'molgenis';
+    flush privileges;
 
-Click: File -> Import ... -> Existing Maven Projects.
+## 3. Configure Eclipse
 
-Set root directory to your git clone folder. E.g. ~/git (this means you can still see the 'molgenis' folder).
+start Eclipse from the commandline (via Dock will not work!)
```

Comment:
Starting via dock doesn't seem to be a problem (any more), at least not on our machines